modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java

```diff
@@ -1876,13 +1875,14 @@ private boolean _isValidImageMimeType(FileEntry fileEntry) {
 		Set<String> extensions = MimeTypesUtil.getExtensions(
 			fileEntry.getMimeType());
 
-		if (Stream.of(
-				_blogsFileUploadsConfiguration.imageExtensions()
-			).anyMatch(
-				extension ->
-					extension.equals(StringPool.STAR) ||
-					extensions.contains(extension)
-			)) {
+		for (String extension :
+				_blogsFileUploadsConfiguration.imageExtensions()) {
+
+			if (!(extension.equals(StringPool.STAR) ||
```

Comment:
@JanisZhang You can just flip the condition here and `return true` in the if-body. It's a very simple one, no need to do shortcut.